Dockerfile

```diff
@@ -8,13 +8,12 @@ RUN --mount=type=cache,target=/var/cache/apt,sharing=locked \
     --mount=type=cache,target=/var/lib/apt,sharing=locked \
     apt-get update -y && apt-get upgrade -y
 
-WORKDIR /CHANGME-component-name
+WORKDIR /eodhp_utils
 ADD LICENSE.txt requirements.txt ./
-ADD CHANGEME-module-name ./CHANGEME-module-name/
+ADD eodhp_utils ./eodhp_utils/
 ADD pyproject.toml ./
 RUN --mount=type=cache,target=/root/.cache/pip pip3 install -r requirements.txt .
 
-# Change as required, eg
-#  CMD ["gunicorn", "-w", "2", "-b", "0.0.0.0", "-k", "uvicorn.workers.UvicornWorker", "--log-level", "debug", "mymodule.main:app"]
-CMD python -m my.module
+
+# CMD python -m my.module
```

Comment:
Probably better to remove this command instead of commenting out.

But, do we even need a docker file for this repo?

Reply:
Removed